shared/devices/dumb.desktop.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+import Devices from './index.render.desktop'
+import type {Device} from '../constants/types/flow-types'
+
+const dev1: Device = {
```

Comment:
the props we send to our components should be our idealized props and not the output of an rpc. the action/reducer should transform it from this into the clean props